doc/api/http.md

```diff
@@ -1290,8 +1290,8 @@ Example:
 ```js
 const http = require('http');
 const server = http.createServer((req, res) => {
-  const ip = res.socket.remoteAddress;
-  const port = res.socket.remotePort;
+  const ip = req.socket.remoteAddress;
+  const port = req.socket.remotePort;
   res.end(`Your IP address is ${ip} and your source port is ${port}.`);
```

Comment:
I guess `res.socket` and `req.socket` point to same entities internally. Let me check and get back

Reply:
```javascript
const http = require('http');
const server = http.createServer((req, res) => {
  console.log(req.socket === res.socket)
}).listen(3000);
```

```
#node h.js
true
^C
#
```